### DISCUSSION TITLE
Can not use array returns in breadcrumbs

### DISCUSSION THREAD

**Discussion by MrVibe** [saadeghi/daisyui#3074] [Q&A]
While below code is perfectly fine it throws up the error 
---
Type error: Type 'string | Element[]' is not assignable to type 'ReactElement<BreadcrumbsItemProps, string | JSXElementConstructor<any>> | (string & ReactElement<BreadcrumbsItemProps, string | JSXElementConstructor<...>>)'.
  Type 'string' is not assignable to type 'ReactElement<BreadcrumbsItemProps, string | JSXElementConstructor<any>> | (string & ReactElement<BreadcrumbsItemProps, string | JSXElementConstructor<...>>)'.

````
<Breadcrumbs>
{Array.isArray(terms) && terms.length > 0
                ? terms.map((term) => (
                    <Breadcrumbs.Item
                      key={term.term_id}
                      href={`/product/product-type/${term?.slug}`}
                    >
                      {term.name}
                    </Breadcrumbs.Item>
                  ))
                : ""}
</Breadcrumbs>
````

**Comment by saadeghi** [comment]
This is daisyUI repo (CSS library).  

For `react-daisyui` (React component library on top of daisyUI) ask your question on this repo:  
https://github.com/daisyui/react-daisyui
